### ISSUE TITLE
No spelling dictionaries on dev server

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#9270]
### System details

    RStudio Edition : Dev Server on Mac
    RStudio Version : master (8f0abea7613d3f5064b002bd5b3237d9654ee3cd)
    OS Version      : macOS 11.2.3
    R Version       : 4.0.5

### Steps to reproduce the problem
- clone `rstudio/rstudio` repo, install dependencies, etc.
- clean previous state via `rstudio/scripts/clean-rstudio` (not sure if necessary to repo, but I had done it)
- in `src/cpp/build` run `cmake .. -GNinja`
-  `ninja`
- `./rserver-dev`
- in `src/gwt` run `ant devmode`
- load IDE (localhost:8787) and look at Spelling options
- then open a file that triggers spellcheck and look at console output (where `rserver-dev` is running)

#### No dictionaries:
![spell check options screenshot](https://user-images.githubusercontent.com/10569626/116119882-c1ee4580-a673-11eb-80eb-255640d8b1a4.png)

#### Error on console when spellcheck tries to run
```
26 Apr 2021 16:33:51 [rsession-gary] ERROR system error 2 (No such file or directory) [path: /Users/gary/rstudio/dependencies/common/dictionaries/en_US.aff]; OCCURRED AT rstudio::core::Error rstudio::core::spelling::(anonymous namespace)::HunspellSpellChecker::initialize(const rstudio::core::spelling::HunspellDictionary &, const rstudio::core::spelling::IconvstrFunction &) src/cpp/core/spelling/HunspellSpellingEngine.cpp:165; LOGGED FROM: void rstudio::core::spelling::HunspellSpellingEngine::Impl::resetDictionaries(const std::string &) src/cpp/core/spelling/HunspellSpellingEngine.cpp:434
```

### Describe the behavior you expected
Be able to use spellcheck on dev build without any other steps required.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by mikebessuille** [comment]
Dev issue; should not delay the release.

**Comment by adamconroy** [comment]
Good point, I'll keep it on 1.5 now so it's not worried about in the lists.
